sinch/core/clients/sinch_client_configuration.py

```diff
@@ -53,6 +56,25 @@ def __init__(
         else:
             self.logger = logging.getLogger("Sinch")
 
+    def _set_voice_origin(self):
+        if not self._voice_region:
+            self.voice_origin = self._voice_domain.format("calling")
+        else:
+            self.voice_origin = self._voice_domain.format("calling-" + self._voice_region)
+
+    def _set_voice_region(self, region):
+        self._voice_region = region
```

Comment:
It would be good to check the region value against a list of supported regions

Reply:
At first glance I've added mentioned validation, but then I've removed it. Enforcing enums is not an easy problem to tackle (we can revisit ADR-0005). For now I've added a helper enum (not enforced) with supported regions.